File: Leetcode/two_pointers/26_Remove Duplicates from Sorted Array/Solution.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        lastNonDup = 0
        for i in xrange(1, len(nums)):
            if nums[i]==nums[lastNonDup]:
                continue;
            lastNonDup+=1
            nums[lastNonDup] = nums[i]
        return lastNonDup+1

File: Leetcode/two_pointers/26_Remove Duplicates from Sorted Array/Solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int lastNonDup = 0;
        for(int i = 1; i<nums.length; ++i){
            if(nums[i]==nums[lastNonDup])
                continue;
            nums[++lastNonDup] = nums[i];
        }
        return lastNonDup+1;
    }
}
